src/openforms/variables/migrations/0011_migrate_interpolation_format.py

```diff
@@ -0,0 +1,71 @@
+# Generated by Django 3.2.18 on 2023-03-28 12:35
+
+import re
+
+from django.db import migrations
+
+MATCH_SINGLE_BRACKETS = r"{([A-z0-9_]*)}"
+REPL_SINGLE_BRACKETS = r"{{\1}}"
+
+MATCH_DOUBLE_BRACKETS = r"{{([A-z0-9_]*)}}"
+REPL_DOUBLE_BRACKETS = r"{\1}"
```

Comment:
It think `r"\."` should be in the character class too. For components nested in layout components or repeated groups. 

(sorry for suggesting regexp changes in text, GitHub for some reason doesn't give me the suggestion button)